### ISSUE TITLE
Transaction extra has unsupported format

### ISSUE THREAD

**Issue by Revinand** [monero-project/monero#2875]
In the latest version (0.11.1) there are a lot of warnings on `rescan_blockchain` call:

```
monero-wallet-rpc[1467]: 2017-11-28 18:21:47.420#011[RPC0]#011WARN #011wallet.wallet2#011src/wallet/wallet2.cpp:660#011Transaction extra has unsupported format: <...>
```

I understand there is nothing criminal, but it seems that this increases request runtime.

**Comment by stoffu** [comment]
Transaction extra is used for attaching auxiliary data to the transaction such as the transaction public key and the payment ID, see https://github.com/monero-project/monero/blob/master/src/cryptonote_basic/tx_extra.h

The official wallet implements a standard protocol to write/read data to/from the tx extra using predefined prefixes, but anyone can modify the code and implement any sort of custom data format in order to attach other arbitrary data to the transaction.

TLDR; you can just ignore those warnings.

**Comment by moneromooo-monero** [comment]
This was getting triggered for empty additional tx keys.

**Comment by stoffu** [comment]
Aha, makes sense.

**Comment by moneromooo-monero** [comment]
This was fixed for empty additional tx keys, it will be kept otherwise.

+resolved

**Comment by Revinand** [comment]
Still see such messages. Can't understand does it affect `rescan_blockchain` command time execution anyhow? Looks like it does

**Comment by icy-ux** [comment]
Still seeing these messages also. Unclear if it matters.
